docs/machine-learning/resources/transforms.md

```diff
@@ -1,136 +1,185 @@
 ---
-title: Data transforms in ML.NET
```

Comment:
Also, I'd reorganize the doc into the name space..

Eg: 
`OneHotEncodingTransformer` should go in the section for `Microsoft.ML.Transforms.Categorical`, "Categorical".